_layouts/event.html

```diff
@@ -9,8 +9,9 @@
 
   <header class="post-header">
     <h2 class="post-title">{{ page.title }}</h2>
-    <p class="event-meta">{{ page.date | date: "%b %-d, %Y" }}{% if page.author %} • <a href="{{ page.author | downcase | replace: ' ', '-'  | prepend: "/team/" | prepend: site.baseurl | prepend: site.url | strip_newlines }}">{{ page.author }}</a>{% endif %}{% if page.meta %} • {{ page.meta }}{% endif %}</p>
+    <p>{{ page.date | date: "%b %-d, %Y" }} • {{ tags_content }}</p>
```

Comment:
@AnneTee could you please replace this with `&centerdot;` (http://dev.w3.org/html5/html-author/charref for reference)